projects/effortless.md

```diff
@@ -0,0 +1,35 @@
+# Effortless
+
+The Effortless project is ...?
```

Comment:
I don't have good words for a description here. Probably should briefly describe mission and development/stability/supported status.

Reply:
Yep I agree I'll put some thought into this

Reply:
🤔 "The Effortless project includes the Habitat Scaffolding for Chef and InSpec and the things to make using that easy." Or something like that maybe

Reply:
```suggestion
The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
```